app/controllers/alert_controller.rb

```diff
@@ -0,0 +1,11 @@
+class AlertController < ApplicationController
```

Comment:
I wonder if we should avoid the word `Alert` because it tends to be an overused term in the email product. It might be easier to just go with `EmailController` and `/emails` since that's what we're doing here.

Reply:
As you were reviewing this I've changed the wording to use 'message' instead of 'alert', do you think this is better?

Reply:
Unfortunately... that's already taken! (Blame @kevindew :smile:)

We decided to call the models added in https://github.com/alphagov/email-alert-api/pull/927 as messages. I think we can probably just change the wording to `email` here since that's the closest model to the controller.

Reply:
Ah damn, okay dokay sure.